lib/commands/build.dart

```diff
@@ -49,9 +49,9 @@ class BuildTpkCommand extends BuildSubCommand
     argParser.addOption(
       'device-profile',
       abbr: 'p',
-      allowed: <String>['mobile', 'tv', 'common'],
-      help:
-          'The type of device that the app will run on. Choose "common" for IoT (Raspberry Pi) devices.',
+      defaultsTo: 'tv',
+      allowed: <String>['mobile', 'tv', 'common', 'tizen'],
+      help: 'The type of device that the app will run on.',
```

Comment:
How about simply
```suggestion
      help: 'The type of device that the app will run on. Choose "common" for the unified Tizen profile.',
```

Reply:
I mean the "tizen" profile can be removed (replaced by "common"). It may just confuse users.

Reply:
However, since Tizen Studio 5.5 (Tizen 8.0), the name of the unified profile is `tizen`.
https://developer.tizen.org/blog/announcing-tizen-studio-5.5-release?langredirect=1
I think using `common` as an alias(?) for `tizen` causes more confusion.
When in the future is updated tizen version,` common` will removed. However, if we use `common` instead of `tizen`, it will eventually change to `tizen`.

However, since `tizen` is still awkward(?) to use as a profile name, I think we can use `common` instead, as you said. Do you still think it needs to be changed?

Reply:
I think `common` still makes sense since the word "common" is a synonym for "unified".

> Added common unified profile called Tizen

But you're also correct. It was just my opinion.

Reply:
As your opinion, I removed the 'tizen' profile. Because the profile name obtained from the `sdb capability` of 8.0 tizen emulator is still 'common'. At least if these things are neatly cleaned(?), I expect to be able to add a `tizen` profile.